app/routes/download_plot.py

```diff
@@ -0,0 +1,82 @@
+# Add the “download results” button to the web interface.
```

Comment:
This comment could be handled better. First, use the triple quotes ```"""``` comments for docstrings at the top of the files. Second, try to make the docstring more informative about the module, rather than explaining what was the assignment.

Reply:
Done